gatsby-config.js

```diff
@@ -1,3 +1,13 @@
+
```

Comment:
remove the new `.yarn/` folder. I think you might have accidentally created this while doing a `yarn`.

Reply:
might need a hand with this just so I dont delete the wrong thing? 🤷‍♂️